src/blueberrypy/command.py

```diff
@@ -445,6 +483,8 @@ def get_command_parser_and_callback(command):
             doc, callback = create.__doc__, create
         elif command == "console":
             doc, callback = console.__doc__, console
+        elif command == "ishell":
```

Comment:
I'd go for extending existing command with autonegotiation of REPL, available for use. See django-extensions - > shell_plus